org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WTableDirEditor.java

```diff
@@ -1010,7 +1010,7 @@ protected CCacheListener(String tableName, WTableDirEditor editor) {
 		}
 
 		@Override
-		public int reset() {			
+		public int reset(int recordId) {
```

Comment:
hi @CarlosRuiz-globalqss , I think the safe approach is to handle both reset() and reset(recordId).

Reply:
Thanks @hengsin - ready